common/app/model/content/Atoms.scala

```diff
@@ -374,6 +414,39 @@ object Quiz extends common.Logging {
   }
 }
 
+object StoryQuiz {
+  def extractImage(quiz: atomapi.quiz.QuizAtom): Option[String] = for {
+    firstQuestion <- quiz.content.questions.headOption
+    firstAsset <- firstQuestion.assets.headOption
+    json = Json.parse(firstAsset.data)
+    assets = json \ "assets"
+    widths = (assets \\ "width").map(_.as[Int])
+    width140 = widths.indexWhere(_ == 140)
+    url <- (assets(width140) \ "secureUrl").asOpt[String]
```

Comment:
Sth like `implicit val assetData = Json.Reads[AssetData]`?

The problem is it there is no type to derive to, [it is just a flat string](https://github.com/guardian/content-atom/blob/master/thrift/src/main/thrift/atoms/quiz.thrift#L20) in the model.